pkg/chunk/gcp/bigtable_index_client.go

```diff
@@ -34,31 +37,44 @@ type Config struct {
 	Project  string `yaml:"project"`
 	Instance string `yaml:"instance"`
 
-	GRPCClientConfig grpcclient.Config `yaml:"grpc_client_config"`
-
-	ColumnKey      bool
-	DistributeKeys bool
-
+	GRPCClientConfig     grpcclient.Config `yaml:"grpc_client_config"`
+	BackoffConfig        util.BackoffConfig
+	ColumnKey            bool
+	DistributeKeys       bool
 	TableCacheEnabled    bool
 	TableCacheExpiration time.Duration
+	WriteLimit           float64
+	MaxWriteBurstSize    int
 }
 
+var (
+	rateLimitedWriteRequest = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "cortex",
+		Name:      "rate_limited_write_requests_total",
+		Help:      "Total count of rate limited write requests.",
+	}, []string{"name"})
```

Comment:
I would call it `operation` (like we did for DynamoDB) and have values like `write-index` and `write-chunk`.

Reply:
okay!